lib/std/c/openbsd.zig

```diff
@@ -1137,18 +1127,6 @@ pub usingnamespace switch (builtin.cpu.arch) {
             sc_cookie: c_long,
         };
     },
-    .aarch64 => struct {
```

Comment:
the struct `ucontext_t` is valid for openbsd aarch64, and useful for segfault handling